### DISCUSSION TITLE
After set password cannot get in

### DISCUSSION THREAD

**Discussion by spcidealacm** [coder/code-server#1843] [Q&A] [answered]
Hello, I use code-server 3.4.1 and change password in ~/.config/code-server/config.yaml. Then restart.
I cannot get in the code-oss unless I set --auth none.
![image](https://user-images.githubusercontent.com/8594876/85922610-13993880-b8c8-11ea-8aa9-f4afb60bdf91.png)
![image](https://user-images.githubusercontent.com/8594876/85922621-39bed880-b8c8-11ea-90ba-1b4df91a65cf.png)

**Comment by spcidealacm** [comment, marked as answer]
I solved this issue. That's because I set port 80 and then use sudo to run it. At this time It cannot use ~/.config/code-server/config.yaml configuration. please fix this problem.

**Comment by code-asher** [comment]
Ah yeah if you run it as `sudo` then it'll use `/root/.config/code-server/config.yaml` instead of `/home/your-user/.config/code-server/config.yaml` (since the home of the root user is `/root` and we just use the home of whatever user is running code-server).

Maybe if it's running as root we should show the path as `/root/` instead of `~/`.

**Comment by nhooyr** [comment]
> Maybe if it's running as root we should show the path as /root/ instead of ~/.

Agreed 👍

**Comment by spcidealacm** [comment]
Agree, and It would be nice if there were multiple login functions :)
